modules/caddyhttp/reverseproxy/streaming.go

```diff
@@ -215,17 +216,17 @@ func (h Handler) copyBuffer(dst io.Writer, src io.Reader, buf []byte) (int64, er
 				written += int64(nw)
 			}
 			if werr != nil {
-				return written, werr
+				return written, fmt.Errorf("writing: %v", werr)
```

Comment:
```suggestion
				return written, fmt.Errorf("writing: %w", werr)
```

I think this is one area of the code where wrapping the error might be useful in case we want to figure out what specific kind of error it is.

Reply:
I thought you were generally against using `%w` for some reason :thinking: 

No objections from me :+1: